src/core/errors.js

```diff
@@ -0,0 +1,19 @@
+const errCode = require('err-code')
+
+exports.ERR_NOT_INITIALIZED = () => {
+  throw errCode(new Error('not initialized'), 'ERR_NOT_INITIALIZED')
```

Comment:
I agree, some consistency here would be beneficial. Would you mind opening a PR to the coding guidelines so that all the core JS developers can get onboard?

Reply:
Which "coding guidelines"?  :grimacing: 
`README.md` in this repo? [ipfs/community/CONTRIBUTING_JS.md](https://github.com/ipfs/community/blob/master/CONTRIBUTING_JS.md) ? Something else?

Reply:
This one please thank you https://github.com/ipfs/community/blob/master/CONTRIBUTING_JS.md

Reply:
Done: https://github.com/ipfs/community/pull/445 :sparkles: